transformer/compose/v1v2.go

```diff
@@ -93,11 +93,13 @@ func parseV2(path string, interpolate bool) (*project.Project, error) {
 	context.ComposeFiles = []string{path}
 	context.ResourceLookup = new(lookup.FileResourceLookup)
 	//TODO: Check if any variable is mandatory
-	someEnvFilePath := ".env"
+	someEnvFilePath := defaultEnvFile
 	if !common.IgnoreEnvironment {
-		absSomeEnvFilePath, err := filepath.Abs(someEnvFilePath)
+		composeFileDir := filepath.Dir(path)
+		absSomeEnvFilePath := filepath.Join(composeFileDir, someEnvFilePath)
+		_, err := os.Stat(absSomeEnvFilePath)
 		if err != nil {
-			logrus.Errorf("Failed to make the path %s absolute. Error: %q", someEnvFilePath, err)
+			logrus.Debugf("Failed to find the env path %s. Error: %q", someEnvFilePath, err)
 			return nil, err
```

Comment:
Why do we want to return with an error when the .env file is not found?

Reply:
Removed this return.